src/lib/util/monacoExtra.ts

```diff
@@ -535,6 +541,16 @@ export const initEditor = (monacoEditor: typeof Monaco): void => {
         [/,/, 'delimiter.bracket'],
         [/\)/, { next: '@pop', token: 'delimiter.bracket' }],
         [/[^),]/, 'string']
+      ],
+      sankey: [
+        configDirectiveHandler,
+        [/(title)(.*)/, ['keyword', 'string']],
+        [/(accTitle|accDescr)(\s*:)(\s*[^\n\r]+$)/, ['keyword', 'delimiter.bracket', 'string']],
+        [/".*?"/, 'string'],
```

Comment:
The string here doesn't work as expected, I tried changing it to:

```suggestion
        [/"(""|[^"])*"/, 'string'],
```

But it gets glitchy.